### ISSUE TITLE
Callouts in code block should use conum font

### ISSUE THREAD

**Issue by wimdeblauwe** [asciidoctor/asciidoctor-pdf#1723]
I want to use JetBrains' Mono font for my code blocks. However, the glyphs needed for the callouts are not in there.

I configured this in my theme:

```
literal:
  font_color: '#0066a1'
  font_family: JetBrainsMono
code:
  font_family: JetBrainsMono
conum:
  font_family: M+ 1mn
```

This gives me the following error:
```
[ERROR] asciidoctor: WARN: Could not locate the character `?' in the following fonts: JetBrainsMono, M+ 1p Fallback, Noto Emoji
```

The rendering is mostly ok:

![image](https://user-images.githubusercontent.com/1115823/87190969-c8355000-c2f3-11ea-99eb-c6463c07d590.png)

The Mono font is being used, but only the callouts below the source code block use the M+ 1mn font. The callouts in the code itself probably try to use the Mono font. 

Is there a setting somewhere so that the code block also uses the conum font ?

(I am using asciidoctor-maven-plugin 2.0.0 with asciidoctorj-pdf 1.5.3)

**Comment by mojavelinux** [comment]
This is a duplicate of #1611 and has already been resolved in the master branch. It will be available in the 2.0.0 release.

**Comment by wimdeblauwe** [comment]
Ah, ok. Looking forward to that!
